tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalServerBehavior.java

```diff
@@ -354,7 +354,7 @@ public void launchServer( ILaunch launch, String mode, IProgressMonitor monitor
 
             if( port != 80 )
             {
-                url += ":" + port;
+                url += ":" + port + "/web/guest/home";
```

Comment:
Lets make the "url" value to ping be customizable per app server.  So that for tomcat we will continue to just ping "localhost:8080"  

Plus, what if '/web/guest/home' is not the default for teh liferay server?  it is possible to make it use something other than '/web/guest/home'.